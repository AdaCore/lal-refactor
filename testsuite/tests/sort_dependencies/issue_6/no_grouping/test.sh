#!/bin/sh

sort_dependencies -P default.gpr -S main.adb --no-separator
