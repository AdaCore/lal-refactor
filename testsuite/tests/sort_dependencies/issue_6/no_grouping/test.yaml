title: Sort without seperating groups with an empty line

description: sort_dependencies tool test

driver: shell_script
