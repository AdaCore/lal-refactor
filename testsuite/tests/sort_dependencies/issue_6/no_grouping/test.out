main.adb
1:1-5:1 with Ada.Containers;
with Ada.Text_IO;
with System.Address_Operations;



